src/app/component/CartItem/CartItem.component.js

```diff
@@ -287,6 +293,7 @@ export class CartItem extends PureComponent {
                 >
                     <span>{ __('Delete') }</span>
                 </button>
+                { isProductInStock ? (
```

Comment:
Prefer using `if` statements and `renderThing` pattern.

Reply:
Okay